src/app/agent/agent-dashboard/create-booking/create-booking.component.css

```diff
@@ -102,4 +102,8 @@
 .custom-switch:hover .tooltiptext {
   visibility: visible;
   opacity: 1;
+}
+
+.ad-image {
+  background-image: url("../../../../assets/images/ad/miami_ad.JPG");
 }
```

Comment:
What's this for?

Reply:
Good catch, it was for adding the image in a different way but it is no longer being used.